### DISCUSSION TITLE
CSS missing 404

### DISCUSSION THREAD

**Discussion by Warlock667** [firefly-iii/firefly-iii#4811] [Firefly III]
Hi,

I am running Firefly III Version 5.5.11 on Ubuntu 21.04 with MySQL and apache.
As Documentroot I have set the /public Directory.
The applicaton itself works far. I can register a user and login, but all the css is missing.
mod_rewrite is activated.

I'm no css expert, but to me it looks like it's all there in the document root

root@firefly-2:/var/www/html/firefly-iii/public# ls -l
total 116
-rw-r--r-- 1 www-data www-data  8370 May  8 19:35 android-chrome-192x192.png
-rw-r--r-- 1 www-data www-data 24194 May  8 19:35 android-chrome-512x512.png
-rw-r--r-- 1 www-data www-data  7134 May  8 19:35 apple-touch-icon.png
-rw-r--r-- 1 www-data www-data  8222 May  8 19:35 apple-touch-icon-precomposed.png
-rw-r--r-- 1 www-data www-data  1084 May  8 19:35 browserconfig.xml
-rw-r--r-- 1 www-data www-data  1045 May  8 19:35 favicon-16x16.png
-rw-r--r-- 1 www-data www-data  1597 May  8 19:35 favicon-32x32.png
-rw-r--r-- 1 www-data www-data 15086 May  8 19:35 favicon.ico
drwxr-xr-x 3 www-data www-data    20 May  8 19:35 images
-rw-r--r-- 1 www-data www-data  2697 May  8 19:35 index.php
-rw-r--r-- 1 www-data www-data   470 May  8 19:35 manifest.webmanifest
-rw-r--r-- 1 www-data www-data   265 May  8 19:35 mix-manifest.json
-rw-r--r-- 1 www-data www-data  6415 May  8 19:35 mstile-150x150.png
-rw-r--r-- 1 www-data www-data    25 May  8 19:35 robots.txt
-rw-r--r-- 1 www-data www-data  1779 May  8 19:35 safari-pinned-tab.svg
drwxr-xr-x 7 www-data www-data    65 May  8 19:35 v1
drwxr-xr-x 6 www-data www-data    56 May  8 19:35 v2
-rw-r--r-- 1 www-data www-data   914 May  8 19:35 web.config

Nonetheless the site looks like this in my browser:

![firefly](https://user-images.githubusercontent.com/52013314/120069100-6523e780-c084-11eb-8036-e2db5e527b5c.jpg)

Has anyone a hint for me what is wrong with my installation ?
Thanks a lot !

**Comment by JC5** [comment]
In the source, check out what the `<base href` thing says.

**Reply by Warlock667** [reply]
Hi James,

the base href looks good to me, except the trailing "/" after index.php. 

![Unbenannt](https://user-images.githubusercontent.com/52013314/120082576-067e5e00-c0c4-11eb-9384-2d4e2e1d14dd.JPG)

**Reply by JC5** [reply]
It shouldn't have the index.php lol.

**Reply by Warlock667** [reply]
Ok, so we are pretty close to the cause of the problem :-)
Can you tell me where/how base href is set ?

**Reply by JC5** [reply]
Make sure that Apache is using mod_rewrite and that allowoverride is set to all. That will automatically correct the base href.

**Reply by Warlock667** [reply]
Yes! "allowoverride all" was missing.
Thanks a lot, James ! :-)
